pep-0517.txt

```diff
@@ -374,12 +374,14 @@ dictionary provided as an "escape hatch" for users to pass ad-hoc
 configuration into individual package builds. Build backends MAY
 assign any semantics they like to this dictionary. Build frontends
 SHOULD provide some mechanism for users to specify arbitrary
-string-key/string-value pairs to be placed in this dictionary. For
-example, they might support some syntax like ``--package-config
-CC=gcc``. Build frontends MAY also provide arbitrary other mechanisms
-for users to place entries in this dictionary. For example, ``pip``
-might choose to map a mix of modern and legacy command line arguments
-like::
+string-key/string-value pairs to be placed in this dictionary.
+In case a user provides duplicate string-key’s, build frontends
+SHOULD combine the corresponding string-value’s into a list of
+string-values. For example, they might support some syntax like
+``--package-config CC=gcc``. Build frontends MAY also provide
+arbitrary other mechanisms for users to place entries in this
+dictionary. For example, ``pip`` might choose to map a mix of
+modern and legacy command line arguments like::
```

Comment:
```suggestion
For example, they might support some syntax like ``--package-config CC=gcc``. 
In case a user provides duplicate string-keys, build frontends SHOULD
combine the corresponding string-values into a list of strings.
Build frontends MAY also provide arbitrary other mechanisms
for users to place entries in this dictionary. For example, ``pip``
might choose to map a mix of modern and legacy command line arguments
like::
```


* Avoid splitting the example from the thing it's providing an example of by moving the inserted line down one sentence (currently, the ordering implies that its an example of a duplicate string key, which it's not).
* Clarify that we're referring to the combined data type ("list of strings"), rather than just repeating "string-values" (which is equivalent to just stating "into a list", and less clear in practice and more repetitive).
* Elide apparently misplaced apostrophes. [^1]
* Rewrap to minimize overall diff

[^1]: If there is a strong reason to separate the noun (which is not, as far as I can tell, a bespoke defined term or a literal somewhere) and the plural, it is not clear to me why, which likely means it would not be clear to most readers either, and thus you're likely better off explicitly stating the distinction anyway as opposed to loosely implying it with the ``.``.

Reply:
Thanks @CAM-Gerlach, applied your suggestion.